Watchman.Discord/Areas/Responses/Services/ResponsesMessageService.cs

```diff
@@ -0,0 +1,93 @@
+        private readonly EmbedMessageSplittingService _embedMessageSplittingService;
+
+        public ResponsesMessageService(ResponsesGetterService responsesDatabase, EmbedMessageSplittingService embedMessageSplittingService)
+        {
+            _responsesDatabase = responsesDatabase;
+            _embedMessageSplittingService = embedMessageSplittingService;
+        }
+
+        public async Task PrintResponses(string commandArgument, ulong serverId)
+        {
+            if (commandArgument == "default")
+            {
+                await _embedMessageSplittingService.SendEmbedSplitMessage("Domyślne responses:", DESCRIPTION, GetDefaultResponses());
+            }
+            else if (commandArgument == "custom")
+            {
+                await _embedMessageSplittingService.SendEmbedSplitMessage("Nadpisane responses:", DESCRIPTION, GetCustomResponses(serverId));
+            }
+            else
+            {
+                await _embedMessageSplittingService.SendEmbedSplitMessage("Wszystkie responses:", DESCRIPTION, GetAllResponses());
+            }
+        }
+
+        private IEnumerable<KeyValuePair<string, string>> GetDefaultResponses()
+        {
+            var responses = _responsesDatabase.GetResponsesFromBase()
+                            .Where(x => x.IsDefault)
+                            .ToList();
+
+            foreach (var response in responses)
+            {
+                yield return new KeyValuePair<string, string>(response.OnEvent, GetMessageWithoutFramesAndBold(response.Message));
+            }
+        }
+
+        private IEnumerable<KeyValuePair<string, string>> GetCustomResponses(ulong serverId)
+        {
+            var responses = _responsesDatabase.GetResponsesFromBase()
+                            .Where(x => !x.IsDefault)
+                            .Where(x => x.ServerId == serverId)
+                            .ToList();
+
+            if (responses.Any() == false)
+            {
+                yield return new KeyValuePair<string, string>("-------------------------------", "```Obecnie jest brak nadpisanych responses!```");
+            }
+            foreach (var response in responses)
+            {
+                yield return new KeyValuePair<string, string>(response.OnEvent, GetMessageWithoutFramesAndBold(response.Message));
+            }
+        }
+
+        private IEnumerable<KeyValuePair<string, string>> GetAllResponses()
+        {
+            var responses = _responsesDatabase.GetResponsesFromBase().ToList();
+
+            foreach (var response in responses)
+            {
+                yield return new KeyValuePair<string, string>(response.OnEvent, GetMessageWithoutFramesAndBold(response.Message));
+            }
+        }
+
+        private string GetMessageWithoutFramesAndBold(string message)
+        {
+            message = message.Contains("`")
+                      ? message.Insert(message.IndexOf("`"), "\\")
```

Comment:
nie prościej zrobić prostego replace z (znak tyldy, github nie wykrywa)
na (ukośnik i znak tyldy, github nie wykrywa)
?

plus
do pojedynczych znaków lepiej używać ' a nie " 
np 'a'
' to typ char
" to typ string => tablica charów
po co ci tablica, jak masz jeden element? 

plus masz takie coś jak @, czyli "traktuj dosłownie", żeby nie pisać \\ jak wystarczyłoby \

Reply:
1. Replace zamienia wszystkie wystąpienia, a mi zależy żeby było tylko przed pierwszym. Wcześniej próbowałem przed wszystkimi, ale później te ukośniki były wszędzie widziane.
2. ' zamiast " - ok
3. w tym przypadku nie widzę jakieś wielkiej różnicy między @"\" a "\\". Do takich krótkich stringów nie ma to znaczenia.